src/lib/formats/rpk.h

```diff
@@ -0,0 +1,149 @@
+// license:LGPL-2.1+
+// copyright-holders:Michael Zapf
+/***************************************************************************
+
+	rpk.h
+
+	RPK format support
+
+***************************************************************************/
+
+#ifndef MAME_FORMATS_RPK_H
+#define MAME_FORMATS_RPK_H
+
+#pragma once
+
+#include "unzip.h"
+#include "hash.h"
+#include "xmlfile.h"
```

Comment:
Some of these are in the .cpp file as well – are they needed here?  They should only be in one of the two places.  Also, keep them sorted.